var/spack/repos/builtin/packages/py-langcodes/package.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyLangcodes(PythonPackage):
+    """Tools for labeling human languages with IETF language tags"""
+
+    homepage = "https://github.com/rspeer/langcodes"
+    pypi = "langcodes/langcodes-3.3.0.tar.gz"
+
+    version("3.3.0", sha256="794d07d5a28781231ac335a1561b8442f8648ca07cd518310aeb45d6f0807ef6")
+
+    depends_on("python@3.6:", type=("build", "run"))
```

Comment:
spack does not support python@3.6 and older, so this line is not needed. You can remove it if you want to make it simpler.